runtime/src/app/app.ts

```diff
@@ -201,7 +203,7 @@ export async function hydrate_target(dest: Target): Promise<HydratedTarget> {
 	};
 
 	if (!root_preloaded) {
-		const root_preload = root_comp.preload || (() => {});
```

Comment:
I changed `() => {}` to `() => ({})`, in other words from a `void` return type to an empty object. If our types are correct that must have been what's intended. I hope it's correct...